sphinxcontrib/confluencebuilder/storage/translator.py

```diff
@@ -2855,6 +2855,37 @@ def visit_vimeo(self, node):
     def visit_youtube(self, node):
         self._visit_video(node, 'https://www.youtube.com/watch?v=' + node['id'])
 
+    # ---------------------------------------------------
+    # sphinx -- extension (third party) -- sphinx-video
+    # ---------------------------------------------------
+
+    def visit_video_node(self, node):
+        autoplay = node.get("autoplay")
+        height = node.get("height")
+        width = node.get("width")
```

Comment:
The only thing I possibly see missing is handling of various height/width unit types. Under the `_visit_video` translation, the implementation utilizes `convert_length` calls to help convert Sphinx-supported units into Confluence-supported units. My guess is we would want some similar logic here as well (unless this extension on accepts a single unit type that matches Confluence's).